src/main/java/jnr/ffi/LibraryLoader.java

```diff
@@ -165,7 +165,7 @@ public static boolean saveError(Map<LibraryOption, ?> options, boolean methodHas
             }
         }
 
-        for (Map.Entry<LibraryOption, ?> option : libraryOptions.entrySet()) {
+        if (libraryOptions != null) for (Map.Entry<LibraryOption, ?> option : libraryOptions.entrySet()) {
```

Comment:
Personally I think this form is fine but we should probably move the for loop within an `if {` block rather than having it inline.